docs/conf.py

```diff
@@ -72,13 +56,35 @@
     'sphinxcontrib.httpdomain',
     'sphinxcontrib.autohttp.flask',
     'sphinxcontrib.autohttp.flaskqref',
+    'sphinxcontrib.apidoc',
 ]
 
 autodoc_default_flags = [
     # 'members',
     # 'undoc-members',
 ]
 
+MOCK_MODULES = [
```

Comment:
Can't we use the built-in mocker now?

Reply:
The built-in mocker behaves differently somehow, in a way I can't quite figure out.